### DISCUSSION TITLE
Keyboard PoC

### DISCUSSION THREAD

**Discussion by joelsernamoreno** [hathach/tinyusb#574] [General]
Hello!

Thanks for the development of TinyUSB

Can you give me the example of a keyboard and the diagram of connections with USB-DIP? I think I need a 33 ohm resistor, but I can't find the connections (USB pins)

I want to play your demo: https://twitter.com/tinyusb/status/1247559949929967617?s=19

Thanks!

**Comment by hathach** [comment]
here is the example https://github.com/hathach/tinyusb/tree/master/examples/device/hid_composite_freertos . For connection wiring, check out the board manual for detail.
